sqlite.go

```diff
@@ -732,6 +755,7 @@ type rows struct {
 	stmt   *stmt
 	closed bool
 	cancel context.CancelFunc // call when query ends
+	done   chan struct{}      // closed when cancellation is done
```

Comment:
not exactly true in the no-context-key case?  then it's closed immediately.

But could also make this nil to mean no need to wait, like above?

Reply:
Done, and so-documented.